packages/jest-cli/src/watch.js

```diff
@@ -225,6 +228,9 @@ const watch = (
           {header: activeFilters(argv)},
         );
         break;
+      case KEYS.R:
+        openReportInBrowser(initialGlobalConfig.coverageDirectory);
+        break;
```

Comment:
We could detect if the coverage directory is actually there and try to open only then.